R/drive_cp.R

```diff
@@ -0,0 +1,118 @@
+#' Copy a Google Drive file.
+#'
+#' `drive_copy()` is an alias for `drive_cp()`.
```

Comment:
I've been removing the square brackets (and, therefore, linkiness) when referring to `foo()` inside the help for `foo()` itself. Because the link does nothing.